notebooks/examples/3_Exploring_results.ipynb

```diff
@@ -44,30 +44,23 @@
    "source": [
-    "## quick scenario run "
+    "## Quick scenario run "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [
     {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "2020-08-19 13:45:55.952 | DEBUG    | subtest.scenario:__init__:54 - Dataset selected: mnist\n",
-      "2020-08-19 13:45:55.953 | DEBUG    | subtest.scenario:__init__:89 - Computation use the full dataset for scenario #1\n",
-      "2020-08-19 13:45:56.014 | INFO     | subtest.scenario:__init__:281 - ### Description of data scenario configured:\n",
-      "2020-08-19 13:45:56.014 | INFO     | subtest.scenario:__init__:282 -    Number of partners defined: 3\n",
-      "2020-08-19 13:45:56.014 | INFO     | subtest.scenario:__init__:283 -    Data distribution scenario chosen: random\n",
-      "2020-08-19 13:45:56.014 | INFO     | subtest.scenario:__init__:284 -    Multi-partner learning approach: fedavg\n",
-      "2020-08-19 13:45:56.019 | INFO     | subtest.scenario:__init__:285 -    Weighting option: uniform\n",
-      "2020-08-19 13:45:56.020 | INFO     | subtest.scenario:__init__:286 -    Iterations parameters: 20 epochs > 3 mini-batches > 8 gradient updates per pass\n",
-      "2020-08-19 13:45:56.020 | INFO     | subtest.scenario:__init__:292 - ### Data loaded: mnist\n",
-      "2020-08-19 13:45:56.020 | INFO     | subtest.scenario:__init__:293 -    48000 train data with 48000 labels\n",
-      "2020-08-19 13:45:56.021 | INFO     | subtest.scenario:__init__:294 -    12000 val data with 12000 labels\n",
-      "2020-08-19 13:45:56.021 | INFO     | subtest.scenario:__init__:295 -    10000 test data with 10000 labels\n"
+     "ename": "NameError",
+     "evalue": "name 'Path' is not defined",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
+      "\u001b[1;32m<ipython-input-1-bca88f8eb45f>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      7\u001b[0m \u001b[0mscenario_params\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m'minibatch_count'\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;36m3\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      8\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 9\u001b[1;33m \u001b[0mexperiment_path\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mPath\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34mr\"C:\\GitHub\\distributed-learning-contributivity\\experiments\\MNIST_experiment\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m     10\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m     11\u001b[0m current_scenario = Scenario(\n",
+      "\u001b[1;31mNameError\u001b[0m: name 'Path' is not defined"
```

Comment:
Sure you want to commit a traceback?

Reply:
Not quite sure after all

Reply:
You'd need to re-run the notebook and to commit it only once it has no error